src/PKSim.Presentation/Presenters/PopulationAnalyses/PopulationAnalysisPKParameterFieldsPresenter.cs

```diff
@@ -59,7 +59,7 @@ private PopulationAnalysisPKParameterField createFieldFrom(QuantityPKParameter p
       {
          var quantity = _allQuantities[pkParameter.QuantityPath];
          if (quantity == null)
-            return null;
+            return _populationAnalysisFieldFactory.CreateFor(pkParameter, QuantityType.Drug | QuantityType.Observer, quantityDisplayPath);
```

Comment:
I'm not sure here. Clearly the quantity is not going to be found in the _allQuantities cache. But I'm not sure how get around this. Should there be additional values added int _allQuantities?

Reply:
I think this is ok but this requires a comment explaining that this parameter is not found because it is a global PK Parameter (can we actually check if this is one?). It cil;d also be the the quantity was removed somehow (thinking about mobi models imported etc..)